CONTRIBUTING.md

```diff
@@ -27,11 +27,11 @@ The [git-scm.com](https://git-scm.com/) site has links to download Git for your
 
 ### Where to get Node.js
 
-You can install Node.js + npm using a [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm) (recommended).
+You can install Node.js [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm) (recommended).
```

Comment:
Technically, installing Node separately isn't necessary anymore.[ This can be managed by pnpm itself](https://pnpm.io/blog/2021/12/29/yearly-update#managing-nodejs-versions-since-v6120), but I'm leaving this here for now. What do you think?